plugins/mac-avcapture/av-capture.mm

```diff
@@ -593,6 +597,52 @@ static inline bool update_frame(av_capture *capture, obs_source_frame *frame,
 	return true;
 }
 
+static inline bool update_audio(obs_source_audio *audio,
+				CMSampleBufferRef sample_buffer)
+{
+	size_t requiredSize = 0;
+	OSStatus status = CMSampleBufferGetAudioBufferListWithRetainedBlockBuffer(
+		sample_buffer, &requiredSize, nullptr, NULL, nullptr, nullptr,
+		kCMSampleBufferFlag_AudioBufferList_Assure16ByteAlignment,
+		nullptr);
+
+	if (status != noErr) {
+		blog(LOG_WARNING,
+		     "[mac-avcapture]: Error while getting size of sample buffer");
+		return false;
+	}
+
+	AudioBufferList *list = (AudioBufferList *)bmalloc(requiredSize);
+	CMBlockBufferRef buffer = NULL;
+
+	status = CMSampleBufferGetAudioBufferListWithRetainedBlockBuffer(
+		sample_buffer, nullptr, list, requiredSize, nullptr, nullptr,
+		kCMSampleBufferFlag_AudioBufferList_Assure16ByteAlignment,
+		&buffer);
+
+	if (status != noErr || !list) {
+		if (list)
+			bfree(list);
+		blog(LOG_WARNING,
```

Comment:
Either separate the `if` contents and the following line with a blank line or wrap the `if` contents in braces.